docs/jwst/associations/association_reference.rst

```diff
@@ -82,80 +84,86 @@ These are the methods of an association rule deal with creation or returning the
 
 
 Creation
 ^^^^^^^^
 
-To create an association based on a member, the ``create`` method of the
-rule is called::
+To create an association based on a member, the :meth:`~jwst.associations.association.Association.create` method of the
+rule is called:
+
+.. code-block:: python
 
-  (association, reprocess_list) = Asn_SomeRule.create(member)
+    association, reprocess_list = Asn_SomeRule.create(member)
 
 ``create`` returns a 2-tuple: The first element is the association and the
-second element is a list of ``reprocess`` instances.
+second is a list of ``reprocess`` instances.
 
 If the member matches the conditions for the rule, an association is
 returned. If the member does not belong, ``None`` is returned for the
 association.
 
-Whether an association is created or not, it is possible a list of
+Whether an association is created or not, a list of
 ``reprocess`` instances may be returned. This list represents the
-expansion of the pool in :ref:`member-with-lists`
+expansion of the pool in :ref:`member-with-lists`.
 
 Addition
 ^^^^^^^^
 
-To add members to an existing association, one uses the :py:meth:`Association.add
-<jwst.associations.Association>` method::
+To add members to an existing association, one uses the :meth:`~jwst.associations.association.Association.add` method:
+
+.. code-block:: python
 
-  (matches, reprocess_list) = association.add(new_member)
+    matches, reprocess_list = association.add(new_member)
 
 If the association accepts the member, the ``matches`` element of the
 2-tuple will be ``True``.
 
 Typically, one does not deal with a single rule, but a collection of
-rules. For association creation, one typically uses an
-:py:class:`~jwst.associations.AssociationRegistry` to collect all the rules a pool will be
+rules. For association creation, one therefore uses an
+:class:`~jwst.associations.AssociationRegistry` to collect all the rules a pool will be
 compared against. Association registries provide extra functionality to
 deal with a large and varied set of association rules.
 
 Saving and Loading
 ^^^^^^^^^^^^^^^^^^
 
 Once created, an association can be serialized using its
-:meth:`Association.dump <jwst.associations.association.Association.dump>` method.
-Serialization creates a string representation of the association which
+:meth:`~jwst.associations.association.Association.dump` method.
+Serialization creates a string representation of the association, which
 can then be saved as one wishes. Some code that does a basic save
-looks like::
+looks like this:
 
-  file_name, serialized = association.dump()
-  with open(file_name, 'w') as file_handle:
-      file_handle.write(serialized)
+.. code-block:: python
 
-Note that ``dump`` returns a 2-tuple. The first element is the suggested
+    file_name, serialized = association.dump()
+    with open(file_name, 'w') as file_handle:
+        file_handle.write(serialized)
+
+Note that ``dump`` returns a 2-tuple: The first element is the suggested
```

Comment:
Question: Why return a suggested filename? What if user choose not to use the suggested filename, will bad things happen?